charts/karpenter/templates/poddisruptionbudget.yaml

```diff
@@ -11,6 +11,11 @@ metadata:
   {{- end }}
 spec:
   maxUnavailable: {{ .Values.podDisruptionBudget.maxUnavailable }}
+  {{- if (semverCompare ">= 1.27-0" .Capabilities.KubeVersion.Version) }}
```

Comment:
Can we use .Capabilities.KubeVersion.Minor? Don't think we care too much about major or patch, right?